### ISSUE TITLE
Automatic Message after Slow Mode

### ISSUE THREAD

**Issue by Randomblock1** [Aliucord/plugin-requests#277]
### Discord Account

Randomblock#0001

### Plugin Description

The plugin will automatically send a message once the slow mode cooldown is over. 

### Go into more detail...

I don't know if it's better as a slash command or if the user should just long press the send icon or something. Also, there should probably be some indicator when it's active. Maybe the Send button changes color or has the countdown instead of the send icon?

Once it's done, adding a message queue would be very helpful too.

### Request Agreement

- [X] I did indeed check to make sure my plugin request is original!

**Comment by nattadasu** [comment]
Huh... I believe automating the message to send using code even it's user input previously) is still counted as selfbotting as per Discord ToS, and I do believe if Aliucord also disagree with selfbot (similar rule to Powercord).

I don't think if automate send button after cooldown time over is ethical from Discord and Aliucord POVs in such way. Also, isn't kinda more like, user will have more complains about "message isn't completely written but message has been sent" than without code? 👀

Moving on... If you want to change the color for send icon, that could break (or not) the theming system. But if it's working as intended and didn't break other plugins.

In short, I don't this could be an effective solution. 

*CMIIW*

**Comment by Randomblock1** [comment]
Huh, I never though about it being self botting. A notification when cool down is over would do basically the same thing, though. Just type a message, wait for it to notify you, then press send. Nothing else would need to be modified. Maybe the notification has a Quick Send button to send what you've already typed, but even without that it would be useful.

**Comment by hairyfred** [comment]
issue with this is
If slowdown is 5 seconds, easy
if its once per hour
if the app is closed in background etc it could be diffucult to do this
On the discord tos/selfbotting side, its on the border of it, id rather let a dev decide this than me

**Comment by Randomblock1** [comment]
I didn't even think about having it notify while closed, I meant the in-app banner thing. But if it's possible, that'd be pretty cool.
But even with the theoretical Quick Send button, it requires user interaction, so it shouldn't count as self botting.
